Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/OpenSAML2ComponentBuilder.java

```diff
@@ -254,8 +169,8 @@ public static OpenSAML2ComponentBuilder getInstance() {
      * @return an Authn Statement
      */
     @Override
-    public AuthnStatement createAuthenicationStatements(final String cntxCls, final String sessionIndex, final DateTime authInstant,
-            final String inetAddr, final String dnsName) {
+    public AuthnStatement createAuthenicationStatements(final String cntxCls, final String sessionIndex,
```

Comment:
Since this file's already being edited, any chance we can fix the typo here? (Authenication -> Authentication)